File: Junk-Codes/Ruby/Scraping/scraping_sample.rb
require 'net/http'
require 'uri'
require 'nokogiri'

uri = URI.parse("https://example.com")
sleep 1
response = Net::HTTP.get_response(uri)
html = response.body

doc = Nokogiri::HTML(html)
h1_content = doc.css('h1')&.text
puts h1_content

File: Junk-Codes/Python/Scraping/scraping_sample.py
import time
import requests
from bs4 import BeautifulSoup

url = "https://example.com"
time.sleep(1)
response = requests.get(url)
html = response.text

soup = BeautifulSoup(html, "html.parser")
h1_content = soup.find("h1").text if soup.find("h1") else None
print(h1_content)
